wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -251,6 +259,64 @@ pub enum WalletCommand {
         utxo_states: Vec<CliUtxoState>,
     },
 
+    #[clap(name = "standalone-add-watch-only-address")]
+    AddStandaloneKey {
+        /// The new standalone watch only address to be added to the selected account
+        address: String,
+
+        /// Optionally specify a label to the new address
+        #[arg(long = "label")]
+        label: Option<String>,
+
+        /// Skip the rescanning of the blockchain
+        #[arg(long = "no-rescan")]
+        no_rescan: Option<bool>,
+    },
+
+    #[clap(name = "standalone-add-private-key-from-hex")]
+    AddStandalonePrivateKey {
+        /// The new hex encoded standalone private key to be added to the selected account
+        hex_private_key: HexEncoded<PrivateKey>,
+
+        /// Optionally specify a label to the new address
+        #[arg(long = "label")]
+        label: Option<String>,
+
+        /// Skip the rescanning of the blockchain
+        #[arg(long = "no-rescan")]
+        no_rescan: Option<bool>,
+    },
+
+    #[clap(name = "standalone-add-multisig")]
```

Comment:
When a public key is invalid, this function doesn't state which public key is invalid.

Reply:
added reporting the index of which address is invalid or not a public key